cohortextractor/query_interface.py

```diff
@@ -4,30 +4,33 @@
 
 class QueryBuilder:
     def __init__(self, table_name):
-        self.table_name = table_name
+        self.table = query_language.table(table_name)
```

Comment:
I think I'm confused again about how we're separating the DSL and the semantic model.  I had thought the idea was to have the tables in `concepts` (or whatever) be separate to the semantic model, which is why it only had a table name before, not a `query_language.table` (which I *think* is a semantic model thing).  

I had been thinking about this one in my head though, and I hasn't come up with an obvious way to get a Variable with enough only non-query_language things which could do all the compiling to query_language things itself

Reply:
I think it makes sense for the objects in `query_interface.py` to not know about semantic model objects until we come to compilation.  So storing a table here is wrong, and we should store the table name.

In particular, this will make serialisation of these DSL objects very easy.